### DISCUSSION TITLE
How to stop it ?

### DISCUSSION THREAD

**Discussion by Menard01** [Motion-Project/motion#1776] [Q&A]
I have used it since years, with Linux Mint, daily, in terminal mode, but now I made a mistake, I updated it ... version proposed is the 4.3.2.1ubuntu0.1 instead of 4.3.2 but after updating I see that the webcam is allways activated and motion is generating some videos as its "conf" file is set, but as I didn't start it. Why ?  It was working perfectly still yesterday.

**Comment by Mr-Dave** [comment]
It is probably running as a service automatically .

`sudo systemctl stop motion` will stop it.
`sudo systemctl disable motion` will disable it from starting again upon reboot.

If you search for `systemctl` you can see the other options.

**Reply by Menard01** [reply]
Thanks, that fixed this problem.
